x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/StringUtils.java

```diff
@@ -57,6 +57,33 @@ public static String camelCaseToUnderscore(String string) {
         }
         return sb.toString().toUpperCase(Locale.ROOT);
     }
+    
+    //CAMEL_CASE to camelCase
+    public static String underscoreToLowerCamelCase(String string) {
```

Comment:
I'm kinda of the fence with this method - maybe it's worth using a different convention but if we stick with it, have an updated comment (the one above is wrong) and some tests.